docs/metrics.md

```diff
@@ -139,6 +204,36 @@ The logs will be printed to stdout in the standard [mozlog format](https://githu
 }
 ```
 
+And Redshift schema:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
+    {"uuid",                       "VARCHAR",   36,      nil,         get_uuid},
+    {"Hostname",                   "VARCHAR",   255,     nil,         "Hostname"},
+    {"logger",                     "VARCHAR",   255,     nil,         "logger"},
+    {"Severity",                   "INTEGER",   nil,     nil,         "Severity"},
```

Comment:
I just copied it from the standard schema for consistency.  it looks inconsistent though in other docs.  I'll change it.